src/main/java/codeu/model/store/basic/ConversationStore.java

```diff
@@ -71,8 +72,9 @@ private ConversationStore(PersistentStorageAgent persistentStorageAgent) {
   }
 
   /** Add a new conversation to the current set of conversations known to the application. */
-  public void addConversation(Conversation conversation) {
+  public void addConversation(Conversation conversation, boolean testing) {
```

Comment:
What's the purpose of these new `testing` parameters?  Are the meant to be permanent or temporary?  Is there another way to achieve the same goal without changing the interface?

Reply:
Whenever a conversation is added into the conversation datastore, it also needs to be added into the activity datastore. However, when testing, the datastores are down (I was getting "errors that said No API available"). So, I decided to add a boolean that indicates if we are testing. I am still trying to find a cleaner way.